os/arch/Kconfig

```diff
@@ -15,13 +15,20 @@ config ARCH_ARM
 	select ARCH_HAVE_CUSTOMOPT
 	---help---
 		The ARM architectures
+config ARCH_XTENSA
```

Comment:
plz  chmod 644 for files

Reply:
Fixed, Thanks.

Reply:
GitHub shows `os/arch/Kconfig 100644 → 100755` now.

Not fixed. Please check a permission again.